src/pty.c

```diff
@@ -431,7 +431,7 @@ mch_isatty(int fd)
     struct strioctl istr;
 
     istr.ic_cmd = ISPTM;
-    istr.ic_timout = 0;
```

Comment:
This is *not* a typo. See [`man stropts.h`](https://manned.org/stropts.h.0p).

Reply:
Thank you @ObserverOfTime!
You are right. https://docs.oracle.com/cd/E88353_01/html/E37842/stropts.h-3head.html

Reply:
Reverted 🍏